doc/nrf/ug_multi_image.rst

```diff
@@ -24,12 +42,134 @@ MCUboot bootloader
+      Multi-image builds are not yet supported for nRF5340.
+
+
+Default configuration
+*********************
+
+The |NCS| samples are set up to build all related images as one solution, starting from the parent image.
+This is referred to as *multi-image build*.
+
+When building the parent image, you can configure how the child image should be handled:
+
+* Build the child image from source and include it with the parent image.
+  This is the default setting.
+* Use a prebuilt HEX file of the child image and include it with the parent image.
+* Ignore the child image.
+
+When building the child image from source or using a prebuilt HEX file, the build system merges the HEX files of the parent and child image together so that they can easily be programmed in one single step.
+This means that you might enable and integrate an additional image without even realizing it, just by using the default configuration.
+
+To change the default configuration and configure how a child image is handled, locate the BUILD_STRATEGY configuration options for the respective child image in the parent image configuration.
+For example, to use a prebuilt HEX file of the :ref:`secure_partition_manager` instead of building it, select :option:`CONFIG_SPM_BUILD_STRATEGY_USE_HEX_FILE` instead of the default :option:`CONFIG_SPM_BUILD_STRATEGY_FROM_SOURCE`, and specify the HEX file in :option:`CONFIG_SPM_HEX_FILE`.
+To ignore an MCUboot child image, select :option:`CONFIG_MCUBOOT_BUILD_STRATEGY_SKIP_BUILD` instead of :option:`CONFIG_MCUBOOT_BUILD_STRATEGY_FROM_SOURCE`.
+
+
+Defining and enabling a child image
+***********************************
+
+You can enable existing child images in the |NCS| by enabling the respective modules in the parent image and selecting the desired build strategy. (? is that correct?)
+To turn an application that you have implemented into a child image that can be included in a parent image, you must add the required configuration options, update the build scripts to make it possible to enable the child image, and make sure that image-specific variables can be disambiguated.
+
+Adding configuration options
+============================
+
+When enabling a child image, you select the *build strategy*, thus how the image is included.
+The three options are:
+
+* Build the child image from source along with the parent image- *IMAGENAME*\_BUILD_STRATEGY_FROM_SOURCE
+* Merge the specified HEX file of the child image with the parent image - *IMAGENAME*\_BUILD_STRATEGY_USE_HEX_FILE, and *IMAGENAME*\_HEX_FILE to specify the HEX file
+* Ignore the child image when building and build only the parent image - *IMAGENAME*\_BUILD_STRATEGY_SKIP_BUILD
+
+You must add these four configuration options to the Kconfig file for your child image, replacing *IMAGENAME* with the (uppercase) name of your child image.
+(? are there any rules for the name? Does it have to be the same as the file name, for example?)
```

Comment:
The name must match the "child image name" mentioned here:

https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/1508/files#diff-99a904e387cfbc7dd8a3cc5e0cf2e7b3R138

Reply:
It does not need to match any file name.

Reply:
Updated to:
You must add these four configuration options to the Kconfig file for your child image, replacing *IMAGENAME* with the (uppercase) name of your child image (as specified in ``zephyr_add_child_image``).

Also, I moved the "Adding configuration options" section after the "Updating the build scripts" section so that it makes sense to refer to that function.